shardingsphere-operator/pkg/controllers/storage_node_controller.go

```diff
@@ -303,28 +334,80 @@ func updateInstanceStatus(node *v1alpha1.StorageNode, instance *rds.DescInstance
 
 func (r *StorageNodeReconciler) reconcileAwsAurora(ctx context.Context, client aws.IRdsClient, node *v1alpha1.StorageNode, dbClass *dbmeshv1alpha1.DatabaseClass) error {
 	// get instance
-	instance, err := client.GetAuroraCluster(ctx, node)
+	aurora, err := client.GetAuroraCluster(ctx, node)
 	if err != nil {
 		return err
 	}
-	if instance == nil {
+	if aurora == nil {
 		// create instance
 		err = client.CreateAuroraCluster(ctx, node, dbClass.Spec.Parameters)
 		if err != nil {
 			return err
 		}
 	}
 	// TODO: update storage node status
+	newStatus, err := updateClusterStatus(ctx, node, client, aurora)
+	if err != nil {
+		return err
+	}
+	node.Status.Cluster = newStatus
+	if err := r.Status().Update(ctx, node); err != nil {
+		r.Log.Error(err, fmt.Sprintf("Failed to update cluster status for node [%s:%s]", node.GetNamespace(), node.GetName()))
+	}
+	r.Recorder.Eventf(node, corev1.EventTypeNormal, "Reconcile", "Reconciled Aurora cluster %s, status is %s", aurora.DBClusterIdentifier, aurora.Status)
+
 	return nil
 }
 
+func updateClusterStatus(ctx context.Context, node *v1alpha1.StorageNode, client aws.IRdsClient, cluster *rds.DescCluster) (v1alpha1.ClusterStatus, error) {
+	clusterStatus := v1alpha1.ClusterStatus{
+		PrimaryEndpoint: v1alpha1.Endpoint{
+			Address: cluster.PrimaryEndpoint,
+			Port:    cluster.Port,
+		},
+	}
+	status := cluster.Status
+	if status == "available" {
+		status = "Ready"
+	}
+	clusterStatus.Status = status
+
+	if len(cluster.ReadReplicaIdentifiers) == 0 {
+		clusterStatus.ReaderEndpoints = []v1alpha1.Endpoint{}
+		return clusterStatus, nil
+	} else {
+
+		for _, readident := range cluster.ReadReplicaIdentifiers {
+			instance, err := client.GetInstanceByIdentifier(ctx, readident)
+			if err != nil {
+				return clusterStatus, err
+			}
+
+			clusterStatus.ReaderEndpoints = append(clusterStatus.ReaderEndpoints, v1alpha1.Endpoint{
+				Address: instance.Endpoint.Address,
+				Port:    instance.Endpoint.Port,
+			})
+		}
+		return clusterStatus, nil
+	}
+}
+
 // deleteDatabaseCluster
 func (r *StorageNodeReconciler) deleteDatabaseCluster(ctx context.Context, node *v1alpha1.StorageNode, databaseClass *dbmeshv1alpha1.DatabaseClass) error {
+	ins, err := aws.NewRdsClient(r.AwsRDS).GetInstance(ctx, node)
```

Comment:
`GetInstance` is used for getting aws rds insatnce, plz set these code in `ase dbmeshv1alpha1.ProvisionerAWSRDSInstance:` space